watt.py

```diff
@@ -26,7 +27,7 @@
 parser.add_argument("-e", "--executor", help="Path to cromwell jar.", default=os.environ.get("EXECUTION_ENGINE"))
 parser.add_argument("--executor-log-prefix",
                     help="Prefix for cromwell log path; outputs will be [flag_input]-[workflow]-[test_name].log.",
-                    default="cromwell ")
+                    default="watt_logs/cromwell")
```

Comment:
One last request: can you also update the README to express what the new default is? Thanks!